README.md

```diff
@@ -36,6 +36,44 @@ Notice BR supports building with Go version `Go >= 1.13`
 
 When BR is built successfully, you can find binary in the `bin` directory.
 
+## Get started in 2 minutes!
+
+```sh
+# Start TiDB cluster
+docker-compose -f docker-compose.yaml rm -s -v && \
+docker-compose -f docker-compose.yaml build && \
+docker-compose -f docker-compose.yaml up --remove-orphans
+
+# Attch to control container runs BR
+docker exec -it br_control_1 bash
+
+# Load testing data to TiDB
+cd /go/src/github.com/pingcap/go-ycsb && \
+make && \
+bin/go-ycsb load mysql -p workload=core \
+    -p mysql.host=tidb -p mysql.port=4000 -p mysql.user=root \
+    -p recordcount=100000 -p threadcount=100
+
+# How many rows do we get?
+mysql -uroot -htidb -P4000 -E -e "SELECT COUNT(*) FROM test.usertable"
```

Comment:
Mention 100000 rows somewhere. The user won't know if getting 0 / 99582 / ... is correct or not.